### ISSUE TITLE
Add a note to the README that this is no longer maintained

### ISSUE THREAD

**Issue by gauntface** [ashi009/node-fast-html-parser#27]
Hi,

I saw on an issue here that this library is no longer maintained, it would be super helpful to add this to the README at the top so people know to search for alternatives and / or know they will need to fork the project if they need any changes.

Thanks for the awesome project,
Matt

**Comment by blex41** [comment]
Agreed. For those still searching, here is a fork which is actively maintained:
https://github.com/taoqf/node-html-parser
